mangaki/mangaki/views.py

```diff
@@ -317,8 +272,8 @@ def get_context_data(self, **kwargs):
 
         if sort_mode == 'mosaic' and not self.is_dpp:
             context['object_list'] = [
-                Work(title='Chargement…', ext_poster='/static/img/chiro.gif')
-                for _ in range(4)
+                (slot_sort_type, Work(title='Chargement…', ext_poster='/static/img/chiro.gif'))
```

Comment:
Can we use `{'sort_type': slot_sort_type, 'work': Work(title=...)}` instead? Would make the template code more readable. BTW we should do `{'work': Work(...)}` for the other cases as well for consistency; that way we can easily get rid of `works_mosaic.html`.